src/main/java/io/xpring/payid/PayIDClient.java

```diff
@@ -71,7 +69,7 @@ public void setEnableSSLVerification(boolean enableSSLVerification) {
    * @return An XRP address representing the given PayID.
    */
   public String xrpAddressForPayID(String payID) throws PayIDException {
-    PaymentPointer paymentPointer = PayIDUtils.parsePayID(payID);
+    PayIDComponents paymentPointer = PayIDUtils.parsePayID(payID);
```

Comment:
Should we still be calling things `paymentPointer`s?

Reply:
For now, spec is uptaken in #196